docs/docs/server/openapi.yaml

```
@@ -38,7 +38,8 @@ paths:
   /conversations/{conversationId}/messages:
     post:
       operationId: addMessage
-      summary: Send a message to the conversation
+      summary: Add message to the conversation
+      description: Add message to conversation and get response from chatbot.
```

```suggestion
      description: Add a message to the conversation and get a response back from chatbot.
```

nit: I think we should use proper sentences for the description since space to display the text isn't really a constraining factor here